data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1010 @@ mission "Unfettered Jump Drive Trading"
+			choice
+				`	(Signal them that I'm going to get the supplies.)`
+				`	(I don't have time for this.)`
+					goto leave
+			label agree
+			`	You are quickly assigned the collection of supplies from <stopovers> and urged to return them as quickly as possible.`
+				accept
+	on stopover
+		dialog `You're surprised to see the Hai are more than happy to give you the medical supplies needed. They load up the cargo and, once they are done, one of them addresses you. "Tell our brothers and sisters to stay strong. Hopefully it is not too late," he says with determination.`
+	to fail
+		has "unfettered: outcast"
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog phrase "generic missing stopover or cargo"
+	on complete
+		event "unfettered: access to Mera's lab"
+		log `Brought medical supplies to Darkcloak to prevent an epidemic.`
+		log "People" "Sookootow" `An Unfettered Chief worried for his people, inside Hai or Unfettered territory. He seems to believe that all Hai are equal, and as a member of the council he has significant influence. He also appears to be quite empathic.`
+		payment 1500000
+		conversation
+			`The staff seem greatly relieved when you return promptly with the medical supplies. To your surprise, some Unfettered have weapons even near this hospital, albeit holstered at all times. As you're returning to your ship, one of them catches up to you. "Hello, <last>. I am Merki, manager of this hospital. I wanted to express my deepest thanks to you and give you the <payment> personally." He takes a second to get his breath back, before continuing, "I find it quite admirable that an outsider would risk getting caught up in all of this. It would mean a great deal to me if your bravery holds fast, so make sure to check the job board for more jobs like this."`
+			`	Once Merki leaves, you notice a recognizable face: the same Hai Chief from Firelode, speaking with someone who, judging by their demeanor and attire, could be the hospital's chief medical officer. He soon sees you as well and hurriedly finishes his conversation, seeming more than a bit emotional as he then moves to speak with you. "I knew you could be counted on as a friend to the Unfettered! With the swift response of captains like you, it seems like the plague has calmed down here. We did not before have the time to get properly introduced. I am Sookootow." He makes a little gesture, you assume of deference, by moving his left hand to the side whilst bowing his head ever so slightly and holding his right hand firmly. You try to imitate the motion, apparently badly, amusing Sookootow. "Worry not, <first> <last>, I already know who you are.`
+			`	"You have been granted access to our basic outfitter on Firelode as a reward for your help, should you need it to store goods or prepare yourself for a raid."`
```

Comment:
You already have access to the outfitter though if you can land…

Reply:
well it's Mera's lab, you certainly can't get in without permission as a human

Reply:
But there is also the normal outfitter that you can already go to.

Reply:
nope no outfitter until this one unlocks
on Firelode

Reply:
True.